lib/text.js

```diff
@@ -1,27 +1,48 @@
-import element from 'magic-virtual-element';
+import React from 'react';
 
-const wrapMark = ({mark, markClass}, el) =>
-  mark ? <mark class={markClass || null}>{el}</mark> : el;
+const wrapMark = ({ mark, markClass }, el) =>
+  (mark ? <mark className={markClass || null}>{el}</mark> : el);
 
-const wrapHref = ({href}, el) => href ? <a href={href}>{el}</a> : el;
+const wrapHref = ({ href }, el) => (href ? <a href={href}>{el}</a> : el);
 
-const wrapBold = ({bold}, el) => bold ? <b>{el}</b> : el;
+const wrapBold = ({ bold }, el) => (bold ? <b>{el}</b> : el);
 
-const wrapItalic = ({italic}, el) => italic ? <i>{el}</i> : el;
+const wrapItalic = ({ italic }, el) => (italic ? <i>{el}</i> : el);
 
-const renderItem = (item) => {
+wrapMark.propTypes = {
+  mark: React.PropTypes.bool,
+  markClass: React.PropTypes.string
+};
+
+wrapHref.propTypes = {
+  href: React.PropTypes.string
+};
+
+wrapBold.propTypes = {
+  bold: React.PropTypes.bool
+};
+
+wrapItalic.propTypes = {
+  italic: React.PropTypes.bool
+};
+
+const renderItem = (item, index) => {
   if (item.type === 'text') {
     let el = String(item.content || '');
     el = wrapItalic(item, el);
     el = wrapBold(item, el);
     el = wrapHref(item, el);
     el = wrapMark(item, el);
 
+    if (React.isValidElement(el)) {
+      el = React.cloneElement(el, { key: index });
+    }
+
     return el;
   }
 
   if (item.type === 'linebreak') {
-    return <br/>;
+    return <br key={index} />;
```

Comment:
@ellell react requires all arrays of elements to have a unique key for each element